### DISCUSSION TITLE
enable automerge on PRs?

### DISCUSSION THREAD

**Discussion by orbeckst** [MDAnalysis/mdanalysis#4605] [Development]
On other repos I started to like [automerge PRs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request). Should we enable it on the MDAnalysis repo?

It looks as if this requires enabling *required status checks* — I think that we had those a while back but apparently not at the moment.

@MDAnalysis/coredevs any opinions?

**Comment by hmacdope** [comment]
Sounds good to me! Anything that makes life easier.

**Comment by orbeckst** [comment]
We _have_ [Allow auto merge](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-auto-merge-for-pull-requests-in-your-repository) already enabled, so presumably the only reason why it doesn't show up is the lack of required status checks.

Then the questions becomes, which (if any) status checks should we be requiring?

**Reply by RMeli** [reply]
I like
- Green CI
- One or more approvals
- All conversations resolved

However, a green CI is often prevented by CodeCov. If we can disable just that one check (I'm not sure it's possible/easy), that would be great.

**Reply by orbeckst** [reply]
We can make the CodeCov status check “not required”.

**Comment by hmacdope** [comment]
We will probably need to choose the least flaky ones, and / or reduce overall flakiness for it to work properly (my experience from other repos)
